lib/ace/http/server.ex

```diff
@@ -116,4 +120,18 @@ defmodule Ace.HTTP.Server do
         {:stop, :normal, {:error, reason}, state}
     end
   end
+  
+  # defp client_info(soc) do
+  #   "soc:\n#{inspect(soc)}\n"
+  # end
+
+  defp client_info({:ssl, socket}) do
```

Comment:
I think this belongs on the `Socket` module and called

Reply:
Ok, that makes sense.